assets/i18n/en.json

```diff
@@ -1,5 +1,40 @@
 {
+    "I18N_ABOUT_PAGE_ABOUT_TAB_CREATE": "Create an Exploration",
+    "I18N_ABOUT_PAGE_ABOUT_TAB_CREATE_TEXT": "about a topic you care about.",
+    "I18N_ABOUT_PAGE_ABOUT_TAB_FEEDBACK": "Earn feedback",
+    "I18N_ABOUT_PAGE_ABOUT_TAB_FEEDBACK_TEXT": "to improve your exploration.",
+    "I18N_ABOUT_PAGE_ABOUT_TAB_HEADING": "About Oppia",
+    "I18N_ABOUT_PAGE_ABOUT_TAB_PARAGRAPH_1": "Oppia's mission is to help anyone learn anything they want in an effective and enjoyable way.",
```

Comment:
In the event where a language's paragraphs don't match English's (ie. A language might be able to accomplish the same meaning in 2 paragraphs whereas English has to have 6 paragraphs or vice versa), it's better for the paragraphs to be explicitly defined (ie. insert `<p>` tags in the JSON) within a larger `18N_ABOUT_PAGE_ABOUT_TAB_PARAGRAPH`.